Update welcome page of the user docs.

deoxys94 opened the pull request:
## Description

- Update the contents to align with the style guide (Remove passive voice and ambiguous pronouns, use a friendly, more conversational tone, shorten sentences to improve readability).
- Change the headers to sentence case.
- Restructure the contents using tables to improve scannability.
- Add short descriptions for some of the missing sections.

### Submitter Checklist

- [ ] Squashed commits with `git rebase -i` (if needed)

davidjharder commented on docs/user/intro.md:
```diff
@@ -6,51 +6,36 @@ sidebar_position: 0
+| [Contributing](/docs/category/contributing)          | Find how you can contribute to Solus, whether you're a beginner or an experienced Linux and Solus user.                                                                                     |
+| [Editions](/docs/user/editions/)                     | Understand the different editions of Solus: differences between editions, configuration tasks, and helpful tips and tricks.                                                                |
+| [Hardware](/docs/user/hardware/)                     | Solus supports most hardware out-of-the-box, but there are exceptions. This section focuses on those exceptions and how to get the most out of your hardware. |
+| [Package management](/docs/user/package-management/) | Learn how to install software and manage the repositories of your Solus system.                                                                                                             |
```
This gives me a 404 when I test locally. I think the trailing slash may need to be removed

davidjharder commented on docs/user/intro.md:
```diff
@@ -6,51 +6,36 @@ sidebar_position: 0
+| [Hardware](/docs/user/hardware/)                     | Solus supports most hardware out-of-the-box, but there are exceptions. This section focuses on those exceptions and how to get the most out of your hardware. |
+| [Package management](/docs/user/package-management/) | Learn how to install software and manage the repositories of your Solus system.                                                                                                             |
+| [Privacy policy](/docs/user/privacy.md)              | Understand our privacy policy: Learn how we handle your data, and how to contact us for privacy concerns.                                                                                   |
+| [Software](/docs/user/software/)                     | Getting started and configuration guides for the software packages included in Solus.                                                                                                       |
```
This gives me a 404 when I test locally. I think the trailing slash may need to be removed

deoxys94 replied:
Fixed

davidjharder reviewed (changes requested):
Thanks for the PR. I like this re-arrangement.
Here are some initial review comments, I'll review the writing a little later.

davidjharder commented:
Writing looks good.

Please run `prettier` to format the tables.

davidjharder commented:
This needs a rebase now.

deoxys94 commented:
Hi @davidjharder,

I fixed the links and rebased the branch.

By the way, why software and package management are under `docs/category/` instead of  `docs/user`? The arrangement seems a little unusual.

davidjharder commented:
> By the way, why software and package management are under `docs/category/` instead of `docs/user`? The arrangement seems a little unusual.

Yeah, it's weird. I don't know docusaurus well enough to do anything other than go along with it for now.

davidjharder reviewed (approved):
Looks good. Thanks for your continued work on the help center
